src/scripts/main.js

```diff
@@ -355,6 +355,25 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+(() => {
+  const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+  if (!dashboard) return;
+
+  const visiblePeople = people.map((person) => ({
+    name: person.name,
+    sex: person.sex,
+    born: person.born,
+    died: person.died,
+    age: person.died - person.born,
+    century: Math.ceil(person.died / 100),
+  }));
```

Comment:
This part of the code could benefit from some comments explaining what's happening. Although the code is quite clear, comments would still be useful, especially for more complex operations. Also, try to avoid using magic numbers like 100. It would be better to create a separate variable for it with a meaningful name.